app/routes/forum/components/ForumDetail.tsx

```diff
@@ -0,0 +1,65 @@
+import { Flex, LoadingIndicator } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import { useParams } from 'react-router-dom';
+import { fetchForum, fetchForums } from 'app/actions/ForumActions';
+import { Content, ContentMain } from 'app/components/Content';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import { selectForumsById } from 'app/reducers/forums';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import ThreadList from './ThreadList';
+
+const ForumDetail = () => {
+  const { forumId } = useParams<{ forumId: string }>();
+  const dispatch = useAppDispatch();
+
+  //This is needed due to action_grant not being sent on detail action for some reason. Should be fixed.
+  usePreparedEffect(
+    'fetchAllForums',
+    () => forumId && dispatch(fetchForums()),
+    [forumId]
+  );
+
+  usePreparedEffect(
+    'fetchDetailForum',
+    () => forumId && dispatch(fetchForum(parseInt(forumId))),
```

Comment:
```suggestion
    () => forumId && dispatch(fetchForum(forumId)),
```
Shouldn't need to parse this. It is just used for the endpoint string